modules/mobile/app/controllers/mobile/v0/check_in_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'chip/service'
+
+module Mobile
+  module V0
+    class CheckInController < ApplicationController
+      def create
+        response = chip_service.post_patient_check_in(appointment_ien: params[:appointmentIEN], patient_dfn:,
+                                                      station_no: params[:locationId])
+        attributes = JSON.parse(response.body)&.dig('data', 'attributes')
+        if (error = parse_error(attributes))
```

Comment:
I didn't want to commit to a full ErrorParser class until we see this in staging and know what the actual responses are going to be